components/properties/src/trievalue.rs

```diff
@@ -22,6 +22,15 @@ impl TrieValue for CanonicalCombiningClass {
     }
 }
 
+impl TrieValue for BidiClass {
+    const DATA_GET_ERROR_VALUE: BidiClass = BidiClass::OTHER_NEUTRAL;
```

Comment:
@echeran ,I added `OTHER_NEUTRAL` for error. Shall I consider different value or create one for errors ?

Reply:
That LGTM.

It's similar to how `WordBreak` handles it. The point is that you want to provide a default value in case of error, so you at least want that value to not convey specific meaning.  Some properties have an "Unknown", some have an "Other".